src/schema/home/home_page_artwork_modules.js

```diff
@@ -26,14 +25,18 @@ const filterModules = (modules, max_rails) => {
   return max_rails < 0 ? allModules : slice(allModules, 0, max_rails)
 }
 
-const addFollowedGenes = (accessToken, modules, max_followed_gene_rails) => {
+const addFollowedGenes = (
+  followedGenesLoader,
+  modules,
+  max_followed_gene_rails
+) => {
   const followedGeneIndex = findIndex(modules, { key: "genes" })
-  if (followedGeneIndex && max_followed_gene_rails !== 1) {
+  if (followedGeneIndex && max_followed_gene_rails >= 1) {
```

Comment:
@alloy I think this may have been a bug introduced in https://github.com/artsy/metaphysics/pull/448 , basically I think this should be `>=`. 

By default, Force uses `1`, and if you have a followed category (and you want to show a max of 1 followed rail) then you _should_ go into this conditional and actually fetch however many followed genes you need.

Currently on production Force homepage (if you have a followed gene) you see:

<img width="737" alt="screen shot 2018-02-28 at 6 11 15 pm" src="https://user-images.githubusercontent.com/1457859/36818407-dce46b60-1cb2-11e8-9805-fb2bc07a534d.png">

This is because the initial call for modules _does_ include an entry for a followed gene rail, yet we wind up not resolving anything here (so the call to fetch the artworks for that module fails in the screenshot above).

Overall I find the homepage code kinda hard to grok, although I feel confident beyond a reasonable doubt ( 🔨 ) that this should be `>=`. For instance, you should be able to disable this rail with a value of `-1`, which currently actually enables it.

I discovered this just by testing all the rails in my local Force -> local Metaphysics were rendering properly in a variety of contexts.

Reply:
Does this still then respect the requirements mentioned in #447?

Reply:
I...believe so? Basically it works the same as before, except because of the `!==` if you specified `1` as the param (which Force uses) it wouldn't work (plus `-1` doesnt disable it).

If you put something thats `>=1`, this update doesn't affect that.

So, I think it would still respect the requirements.